bravado/fido_future.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#
+# Copyright (c) Yelp, Inc.
+#
+
+"""Code for checking the response from API. If correct, it proceeds to convert
+it into Python class types
+"""
+import logging
+
+import fido
+
+from bravado.mapping.http_future import HttpFuture, DEFAULT_TIMEOUT_S
+from bravado.mapping.operation import unmarshal_response
+from bravado.mapping.response import ResponseLike
+
+log = logging.getLogger(__name__)
+
+
+class FidoResponseAdapter(ResponseLike):
+    """Wraps a fido.fido.Response object to provider a uniform interface
+    to the response innards.
+
+    :type requests_lib_response: :class:`fido.fido.Response`
+    """
+    def __init__(self, requests_lib_response):
+        self._delegate = requests_lib_response
+
+    @property
+    def status_code(self):
+        return self._delegate.code
+
+    def json(self, **_):
+        return self._delegate.json()
```

Comment:
but `Fido`'s response doesn't take any params to json() in [fido.py](https://github.com/Yelp/fido/blob/master/fido/fido.py#L41)